#include <exception>
#include "test_utils.hpp"


int main()
{
    test_assert(false);

    return 0;
}